cmd/topical/main.go

```diff
@@ -28,7 +28,12 @@ func main() {
 
 	// Initialize session, HTML templates, and storage interface
 	session := session.NewSession(ac.SessionKey)
-	templates := templates.GenerateTemplates("./web/views/*.gohtml")
+	templates, err := templates.GenerateTemplates("./web/views/*.gohtml")
+
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Should there be an exit clause?

Reply:
`log.Fatal` is basically a `Print()` call followed by a call to `os.Exit(1)`. It's pretty neat [docs](https://golang.org/pkg/log/#Fatal).

Reply:
Cool. Thanks